orderly_web/cli.py

```diff
@@ -0,0 +1,30 @@
+"""Usage:
+  orderly_web start <path>
+  orderly_web status <path>
+  orderly_web stop <path> [--volumes] [--network] [--kill]
+
+Options:
+  --volumes   Remove volumes (WARNING: irrevrsible data loss)
+  --network   Remove network
+  --kill      Kill the containers (faster, but possible db corruption)
+"""
+
+import docopt
+
+import orderly_web
+
+
+def main(argv=None):
+    args = docopt.docopt(__doc__, argv)
+    path = args["<path>"]
+
+    cfg = orderly_web.read_config(path)
+    if args["start"]:
+        orderly_web.deploy(cfg)
```

Comment:
Is there a distinction to be made between `start` i.e. start an existing set of containers and `deploy` i.e. deploy possible new containers? Likewise `stop` i.e. just stop the containers vs say `kill` i.e. `remove the containers`? Maybe unnecessary, but perhaps renaming the cli argument `start` to `deploy` would reflect what's going on a bit better?